src/DataCollector/MultiAuthCollector.php

```diff
@@ -115,7 +115,7 @@ protected function getUserInformation($user = null)
         // The default auth identifer is the ID number, which isn't all that
         // useful. Try username and email.
         $identifier = $user instanceof Authenticatable ? $user->getAuthIdentifier() : $user->id;
```

Comment:
It would also be great to change `$user->id` for `$user->getKey()` 
Fix in case the primary key is not `id`